tests/src/Functional/CommentsGenerationTest.php

```diff
@@ -57,6 +57,21 @@ public function testThereIsNoNewLineAboveTagsSection(\ReflectionClass $mainClass
     {
         $property = $mainClass->getProperty('classmap');
         $this->assertContains('/**
-     * @var array', $property->getDocComment());
+     * @var array', $this->normalizeNEL($property->getDocComment()));
+    }
+
+    /**
+     * Normalizes the newline.
+     *
+     * Windows uses CR+LF for the newline.
+     * This method converts the newline to UNIX style.
+     *
+     * @param string $string
+     *
+     * @return string
+     */
+    private function normalizeNEL($string)
+    {
+        return str_replace("\r", '', $string);
```

Comment:
There issue with the Reflection component, not with git. I'll be yet look to this issue, within days.

*I use `core.autocrlf = input` for all repositories by default.*